scripts/authors.sh

```diff
@@ -1,16 +1,14 @@
 #! /bin/bash
 
-# Usage:
-#   `./authors.sh`
-#		Print a list of all authors who have committed to develop since master.
-#
-#   `./authors.sh <email address>`
-#		Lookup the email address on Github and print the associated username
+ref=$1
```

Comment:
```suggestion
ref="${1:?missing target reference}"
```

or maybe supply a default? E.g., `git tag --list --sort=-authordate | head -1` or similar?

Reply:
```suggestion
set -euo pipefail

ref=$1
```

Reply:
I think I like keeping the usage printed so that people get better sense of how the script is meant to be used.